api/src/test/java/org/openmrs/api/StorageServiceTest.java

```diff
@@ -1,623 +0,0 @@
-/**
```

Comment:
Did you intentionally remove this test class? 😊

Reply:
Yes, it's divided now into LocalStorageServiceTest and S3StorageServiceTest.